examples/checkout/search/search_abandoned_transactions.rb

```diff
@@ -8,7 +8,10 @@
 #   You can pass the credentials parameters to PagSeguro::Transaction#find_abandoned
 
 options = {
-  credentials: PagSeguro::AccountCredentials.new('EMAIL', 'TOKEN')
+  credentials: PagSeguro::AccountCredentials.new('EMAIL', 'TOKEN'),
+  starts_at: (Time.now - 86400),
```

Comment:
seconds…

It can be `Time.now(2016, 1, 13, 14, 17)` or `Time.new(2016, 1, 13, 1, 0)` but the PO wants to "be more self explainable", aka, if I put a "static" time users will not read it and they will just run. (PO just runed it ¬¬')